console/frontend/src/main/frontend/src/app/components/datatable/dt-content.directive.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { DtContentDirective } from './dt-content.directive';
+import { Component, DebugElement, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+@Component({
+  standalone: true,
+  template: `<ng-template appDtContent>TEST</ng-template>`,
+  imports: [DtContentDirective],
+})
+class TestComponent {
+  @ViewChild(DtContentDirective) content!: DtContentDirective<object>;
+}
+
+describe('DtContentDirective', () => {
+  let fixture: ComponentFixture<TestComponent>;
+  // let directiveElement: DebugElement;
+  beforeEach(() => {
+    fixture = TestBed.configureTestingModule({
+      imports: [DtContentDirective, TestComponent],
+    }).createComponent(TestComponent);
+
+    fixture.detectChanges(); // initial binding
+
+    // directiveElement = fixture.debugElement.query(By.directive(DtContentDirective));
```

Comment:
Clean up the comments or maybe state why it is usefull to keep.

Reply:
Cleaned up in https://github.com/frankframework/frankframework/pull/7506/commits/0bad7c58386468d1523484a43ffe4babc0db2f95